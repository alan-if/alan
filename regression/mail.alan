Options
    Debug.


Verb take
    Does
	Locate Object In inventory.
	"Taken."
End Verb take.


Object mailbox Name mail box At l
    Container
End Object.


Object mail Name some mail At l
End Object.


Location l
End Location.


Start At l.
