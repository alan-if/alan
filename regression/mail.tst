../compiler/alan mail
../interpreter/arun -n mail < mail.input
