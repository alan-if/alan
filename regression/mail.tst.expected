########## mail ##########
../compiler/alan mail
../interpreter/arun.exe -n mail < mail.input
########## /bin/sh  mail.tst ##########

        No warnings or errors detected.
        2 informational message(s).


L.
There is a mail box and a some mail here.

> take mail
Taken.

> take mail box
Taken.

> 

Do you want to RESTART, RESTORE or QUIT ? 
